### ISSUE TITLE
[AXE-CORE]: VAOS - Upcoming meeting links MUST be disabled semantically to avoid color contrast error

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#8142]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

While running an axe scan for the upcoming / past appointments, I saw an error about the upcoming (disabled) meeting links not meeting the 4.5:1 color contrast ratio. This is an automated error, so should be fixed as soon as we can scope it. Screenshot and code fix snippet attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Axe errors for links being below 4.5:1 color contrast ratio do not appear on future runs
- [ ] Link styles do not change
- [ ] `aria-disabled="true"` is removed when the meeting invite is "live" and ready to join

## Possible Fixes (optional)

We could fix this with one added line of HTML:

```html
<!-- Add an aria-disabled="true" to these upcoming appointments -->
<!-- Remove it when the meeting is active -->
<a
  aria-describedby="description-join-link-189d685b2dc65719b6501900e390ddc9"
  aria-disabled="true"
  class="usa-button vads-u-margin-left--0 vads-u-margin-right--1p5 usa-button-disabled"
  href="https://UNIQUE_URL_HERE"
  rel="noopener noreferrer"
  target="_blank"
>
  Join session
</a>
```

## Screenshots or Trace Logs

![Screen Shot 2020-04-17 at 12 56 25 PM](https://user-images.githubusercontent.com/934879/79600371-ab023400-80ac-11ea-95e7-0d288abf33c1.png)

**Comment by jbalboni** [comment]
@1Copenut Do you need to verify this on staging before I close this?

**Comment by 1Copenut** [comment]
@jbalboni Yes, I'll run a quick axe check in staging and call it a day.

**Comment by jbalboni** [comment]
@1Copenut Cool, should be on staging as of a few minutes ago. I'm not sure if there are video appointments to view in this state, though, and it may take a while to get them created.

**Comment by 1Copenut** [comment]
I just verified with a test account that has some video links. No more axe error. Moving to close.
